renderers/chords_renderer.js

```diff
@@ -91,28 +92,19 @@ function appendVoiceContentDiv(parentVoice, text, whitespace, className) {
   textDiv.className = className;
 
   if (className.startsWith('c')) {
-    const chordDiagramDiv = document.createElement('div');
-    chordDiagramDiv.className = 'diagram';
-
-    // TODO: Get fingering (or list of fingerings) from chord JSON.
-    const tempSVG = {
-      fretOffset: 2,
-      mutes: [
-        { string: 1 },
-        { string: 2 }
-      ],
-      notes: [
-        { string: 3, fret: 3 },
-        { string: 4, fret: 5 }
-      ],
-      barres: [
-        { first: 5, last: 6, fret: 6 }
-      ]
-    };
-    chordDiagramDiv.innerHTML = chordDiagram.renderChordDiagram(tempSVG);
-
     textDiv.className += ' chord';
-    textDiv.appendChild(chordDiagramDiv);
+
+    if (guitarChordLibrary.has(text.toString())) {
+      const chordDiagramDiv = document.createElement('div');
+      chordDiagramDiv.className = 'diagram';
+
+      const shorthands = guitarChordLibrary.get(text.toString());
+      const svgs = shorthands.map(
+        (shorthand) => chordDiagram.renderChordDiagram(shorthand));
```

Comment:
This could be shortened to:

```javascript
const svgs = shorthand.map(chordDiagram.renderChordDiagram)
```

Reply:
👍

Reply:
Actually, it doesn't work. Turns out a second argument, `index` is passed which messes up the renderer. I think we can modify the interface of renderChordDiagram to make it work.

Reply:
Ah, makes sense. I had tried it with `.forEach`. 👍